kong/reports.lua

```diff
@@ -114,7 +114,7 @@ end
 
 local function send_report(signal_type, t, host, port)
```

Comment:
I don't think it's necessary to change the return value of `send_report`, otherwise these calls below would better be changed

1. https://github.com/Kong/kong/blob/3794d112fde76d2808cf88c0448f5d43771b006a/kong/api/routes/config.lua#L23
2. https://github.com/Kong/kong/blob/3794d112fde76d2808cf88c0448f5d43771b006a/kong/api/routes/plugins.lua#L41
3. https://github.com/Kong/kong/blob/3794d112fde76d2808cf88c0448f5d43771b006a/kong/cmd/config.lua#L131

Reply:
The addition of an error message as the second return value to the `send_report` function does not affect these call sites as don't even use the return value.